src/api/index.tsx

```diff
@@ -87,17 +106,24 @@ export const getVersionFile = async ({
   path,
   versionId,
 }: GetVersionFileParams) => {
-  return callApi({
+  return callApi<ExternalVersion>({
     apiState,
     endpoint: `reviewers/browse/${versionId}`,
     query: path ? { file: path } : undefined,
   });
 };
 
 export const logOutFromServer = async (apiState: ApiState) => {
-  return callApi({
+  return callApi<{}>({
```

Comment:
Hmm, I think it's actually an empty response. Maybe that doesn't matter.

Reply:
Indeed, the API returns a 204 but I didn't know how to express that. Should we use `null`?